Package.swift

```diff
@@ -24,6 +24,12 @@ let package = Package(
       name: "CombineSchedulers",
       dependencies: [
         .product(name: "XCTestDynamicOverlay", package: "xctest-dynamic-overlay")
+      ],
+      swiftSettings: [
+        .unsafeFlags(["-Xfrontend", "-application-extension"])
```

Comment:
While local/branch dependencies can use `unsafeFlags` for the sake of development, SwiftPM will reject a versioned dependency that attempts to use them, so we can't rely on them here.

Can you share a repro? I seem to have no problem creating a new app with an app extension, adding the `combine-scheduler` package, linking it to the extension, and building.

Reply:
I think it depends how the framework is added, i.e., whether it's added as dynamic framework or not.

I created a sample project and added the `CombineSchedulers` dynamically and I can see the warning. But if added as explained in the [Installation](https://github.com/pointfreeco/combine-schedulers#installation) section, the warning would go away.

Reply:
@kasrababaei I'm not quite sure what I'm looking at, but the sample project you provide doesn't build because it relies on a Carthage xcframework product to be built:

> **Warning**: There is no XCFramework found at '/…/neo-ios/Carthage/Build/CombineSchedulers.xcframework'.

In general this sounds like more of a tooling issue that would affect _all_ Swift packages and not a bug with this package. But given that `unsafeFlags` are not safe to use in released packages, I don't think we can take this PR as is. Mind if I close?

Reply:
My bad, I forgot to upload the xcframework file. But I think you're right, it sounds like we should just add `CombineSchedulers` as a static framework. Please close this PR and thank you for the quick response.